src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -19,6 +19,16 @@
 		<StartupObject>Program</StartupObject>
 	</PropertyGroup>
 
+	<ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
+		<None Include="config\netfx\App.config" />
+		<None Remove="config\netcoreapp\App.config" />
+	</ItemGroup>
+
+	<ItemGroup Condition="$(TargetFramework.StartsWith('netcoreapp'))">
```

Comment:
Should we handle .NET 5 now so this works when we upgrade the test project in the future.

Looks like you can use `GetTargetFrameworkIdentifier` which sounds like it'll always return `.NETCoreApp` on .NET Core and .NET 5:
https://docs.microsoft.com/en-us/dotnet/core/compatibility/msbuild/5.0/targetframework-name-change

Reply:
I didn't know about that yet! Done.